apps/dashboard/app/views/active_jobs/index.html.erb

```diff
@@ -59,46 +57,13 @@
   </div>
 </div>
 
-<script>
-var filter_id = <%= (@jobfilter ? "'#{@jobfilter}'" : "localStorage.getItem('jobfilter') || '#{default_filter_id}'").html_safe %>;
-var cluster_id = <%= (@jobcluster ? "'#{@jobcluster}'" : "localStorage.getItem('jobcluster') || 'all'").html_safe %>;
-var performance_tracking_enabled = false;
-
-<% if Configuration.console_log_performance_report? %>
-function report_performance(){
-  var marks = performance.getEntriesByType('mark');
-  marks.forEach(function(entry){
-    console.log(entry.startTime + "," + entry.name);
-  });
-
-  // hack but only one mark for document ready, and rest are draw times
-  if(marks.length > 1){
-    console.log("version,documentReady,firstDraw,lastDraw");
-    console.log(`<%= `git describe --always --tags`.strip() %>,${marks[0].startTime},${marks[1].startTime},${marks.slice(-1)[0].startTime}`);
-  }
-}
-
-performance_tracking_enabled = true;
-performance.mark('document ready - build table and make ajax request for jobs');
-<% end %>
-
-var table = create_datatable({
-  drawCallback: function(settings){
-    // do a performance mark every time we draw the table (which happens when new records are downloaded)
-    if(performance_tracking_enabled && settings.aoData.length > 0){
-      performance.mark('draw records - ' + settings.aoData.length);
-    }
-  },
-  base_uri: '<%= controller.relative_url_root %>'
-});
+<div hidden=true
+  id="active_jobs_config"
+  data-filter-id=<%= (@jobfilter ? "'#{@jobfilter}'" : "localStorage#{default_filter_id}").html_safe %>
+  data-cluster-id=<%= (@jobcluster ? "'#{@jobcluster}'" : "localStorage").html_safe %>
```

Comment:
I wonder if there's a better way to do this than regular expression parsing on this value here. 

Also take care with `html_safe` - are these are query parameters (meaning someone can craft a malicious link)? Though that maybe more of an FYI - I just tried something like this and it doesn't appear to be evaluated, at least on Firefox. So I don't think, because of the placement here in an attribute, that it could actually be evaluated.
```html
      <div data-something="<script>alert('hello');</script>">

      </div>
```